lib/webvalve.rb

```diff
@@ -21,7 +22,8 @@ class << self
     delegate :setup, :register, :whitelist_url, :reset, to: :manager
 
     def enabled?
-      Rails.env.in?(ENABLED_ENVS)
+      Rails.env.in?(ALWAYS_ENABLED_ENVS) ||
+        ENABLED_VALUES.include?(ENV['WEBVALVE_ENABLED'])
```

Comment:
Do you want to allow forced disablement in an always-enabled env? Or perhaps alert when somebody attempts to disable in that context if you don’t want to respect it?